components/molecules/ClearFiltersButton/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import IconComposer from "@rubin-epo/epo-react-lib/IconComposer";
+import styles from "./styles.module.css";
+import classNames from "classnames";
+import { usePathname, useRouter, useSearchParams } from "next/navigation";
+
+interface ClearFiltersButtonProps {
+  className?: string;
+  onFiltersCleared?: () => void;
+}
+
+const ClearFiltersButton: FC<ClearFiltersButtonProps> = ({
```

Comment:
clear filters button molecule that just deletes all the current search params, disabled if no params are set